db/migrate/20191026135203_add_deactivated_at_to_users.rb

```diff
@@ -0,0 +1,5 @@
+class AddDeactivatedAtToUsers < ActiveRecord::Migration[5.2]
+  def change
+    add_column :users, :deactivated_at, :datetime
```

Comment:
It would be good idea to index this column, because we will use `User.active` all over the place to only list active users
`add_column :users, :deactivated_at, :datetime, index: true`